### ISSUE TITLE
Add option to install Meteor on Windows in a custom location

### ISSUE THREAD

**Issue by zarub2k** [meteor/meteor#4426]
I am not able to install Meteor in Windows OS under specific directory. Whenever I click the .exe file the installation is happening somewhere...

How can I change the behavior? I need to install them in a specific directory. Then only I can specify the directory easily in my classpath

Otherwise Meteor command is will not work.

**Comment by stubailo** [comment]
Currently, it's being installed inside `%LOCALAPPDATA%\\.meteor`. Is that not sufficient for your needs?

**Comment by avital** [comment]
We wrote up instructions for how to manually install Meteor on Windows without the installer: https://github.com/meteor/meteor/wiki/Meteor-on-Windows:-Alternate-installation. You can change the directory you extract Meteor into and it should still work.

Hope this helps!

**Comment by DiegoOlivierFernandezPons** [comment]
In most corporate IT managed machines, Users\ has size limitations. I just received an email from IT saying C:\Users\[User]\AppData\Local\.meteor was 3.6Gb and I needed to move it elsewhere. 

- First I tired to just copy .meteor into another location and it failed with "the filenames are too long for the destination folder ... "
- Then I tried to uninstall meteor (to reinstall it manually based on instructions above) but there is no uninstaller. So I manually deleted .meteor and searched in regedit for all the instances of 'meteor' and deleted the correponding keys.
